### DISCUSSION TITLE
Error message

### DISCUSSION THREAD

**Discussion by blogdemoi** [ArchiveBox/ArchiveBox#1136] [Q&A]
Hi,

So now my Archivebox is running fine, thank you. Sometimes I have this kind of error messages:

> Snapshot [[1681128035.017187]](http://10.0.242.215:8005/archive/1681128035.017187/index.html) exists in DB, but resource 1681128035.017187/singlefile.html does not exist in [snapshot dir](http://10.0.242.215:8005/archive/1681128035.017187/) yet.
> 
> Maybe this resource type is not availabe for this Snapshot,
> or the archiving process has not completed yet?
> 
> # run this cmd to finish archiving this Snapshot
> archivebox update -t timestamp 1681128035.017187
> 
> 
> 
> Next steps:
> - list all the [Snapshot files .*](http://10.0.242.215:8005/archive/1681128035.017187/)
> - view the [Snapshot ./index.html](http://10.0.242.215:8005/archive/1681128035.017187/index.html)
> - go to the [Snapshot admin](http://10.0.242.215:8005/admin/core/snapshot/ecac9e39-5804-4266-88b0-a2b711356d7c/change/) to edit
> - go to the [Snapshot actions](http://10.0.242.215:8005/admin/core/snapshot/?id__startswith=ecac9e39-5804-4266-88b0-a2b711356d7c) to re-archive
> - or return to [the main index...](http://10.0.242.215:8005/)

What can I do to prevent this?

Thank you.

**Comment by pirate** [comment]
It's not a critical error, basically it means that Singlefile either hasn't run for that URL url (maybe archiving was interrupted), or it ran and wasn't able to archive the page.

When you run that command it shows does it succesfully complete? If so, that means it was interrupted the first time. Maybe the network request timed out due to some lag, or the process was killed?
If you run the command and it errors out, that means singlefile cant archive that page for some reason. This is normal to see sometimes, not all sites work well with all methods, which is why we try a variety of methods for each site.

If you see these errors happening for more than >20% of your snapshots let me know and I can take a closer look at the specific URLs and error logs.

**Reply by blogdemoi** [reply]
So, now I solved my other problems, I am coming back here because I have errors every time I try to save with Singlefile.

E.g:

>  > singlefile                                          
>         Extractor failed:                                   
>              SingleFile was not able to archive the page
>             Got single-file response code: 0.
>             TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md
>         Run to see full output:
>             cd /data/archive/1684282634.902824;
>             /node/node_modules/single-file/cli/single-file --browser-executable-path=chromium "--browser-args=[\"--headless\", \"--no-sandbox\", \"--disable-gpu\", \"--disable-dev-shm-usage\", \"--disable-software-rasterizer\", \"--run-all-compositor-stages-before-draw\", \"--hide-scrollbars\", \"--user-agent=Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.61 Safari/537.36 ArchiveBox/{VERSION} (+https://github.com/ArchiveBox/ArchiveBox/)\", \"--window-size=1440,2000\"]" https://fasos-research.nl/occupationstudies/collaboration-in-japanese-occupied-china-1937-1945-old-and-new-questions/ singlefile.html

What should I do? Thank you for any suggestions.

**Reply by pirate** [reply]
Can you try with the latest dev branch `archivebox/archivebox:dev`, we've updated chrome to v110+
